gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/environments/domains/CertificatesResource.java

```diff
@@ -110,12 +127,38 @@ public void list(
                             return true;
                         })
                         .map(cert -> this.filterCertificateInfos(cert, certificateExpiryThresholds.get(0)))
+                        .flatMapSingle(this::addApplications)
                         .sorted((o1, o2) -> String.CASE_INSENSITIVE_ORDER.compare(o1.getName(), o2.getName()))
                         .toList()
-                        .map(sortedCertificates -> Response.ok(sortedCertificates).build()))
+                        .map(sortedCertificates -> {
+
+                            // compute RENEWED status for System certificates
+                            sortedCertificates.stream()
+                                    .filter(Certificate::isSystem)
+                                    .sorted(new CertificateTimeComparator())
+                                    .skip(1) // first element is the latest System Certificate thanks to the CertificateTimeComparator
```

Comment:
Perhaps this literal could be a an appropriately named constant? That might remove the need for the comment.